openapi/access-token/server.yaml

```diff
@@ -17,7 +17,6 @@ paths:
         - Client Authorization Service
       summary: Authenticate with the BankAxept ePayment system and receive an Authorization token.
       requestBody:
-        required: false
```

Comment:
this defaults to false. So ideally we would set this to required, and also require the "grantType". But it doesn't work with our API, and I think it's related to spring webflux not being able to take in these kinds of parameters directly, you have to get them through `serverwebexchange getformdata`